src/main/java/TimedTask.java

```diff
@@ -0,0 +1,30 @@
+public abstract class TimedTask extends Task {
+    private final String timing;
+
+    public TimedTask(String taskName, String timing) {
+        super(taskName);
+        this.timing = timing;
+    }
+
+    public static class Deadline extends TimedTask {
+        public Deadline(String taskName, String timing) {
+            super(taskName, timing);
+        }
+
+        @Override
+        public String status() {
```

Comment:
Would it be better if you used verbs to name methods like these, eg. `showStatus()` instead of `status()`?